src/lib/types/metadata.ts

```diff
@@ -0,0 +1,21 @@
+export interface Attribute {
+	trait_type: string;
+	value: string | number;
+}
+
+interface Meta {
+	[key: string]: any | undefined;
+	image: string;
+	animation_url?: string;
+	isBiddable?: boolean;
+	gridViewByDefault?: boolean;
+	customDomainHeader?: boolean;
+	previewImage?: string;
+	customDomainHeaderValue?: string;
+	image_full?: string;
+	attributes?: Attribute[];
+}
+
+export interface Metadata extends Meta {
+	title: string;
+}
```

Comment:
Can you remember why we set it up like this? If yes, can you add some comments so the next person to look at it knows what's going on?